doc/source/drivers/raster/zarr.rst

```diff
@@ -505,6 +505,9 @@ The following options are creation options of the classic raster API only:
       When writing a multi-band dataset as a 3D Zarr array, whether the band
       dimension should be the first one/slowest varying one (BAND), or the
       last one/fastest varying one (INTERLEAVE)
```

Comment:
small addition: end of sentence should be (PIXEL)

Reply:
fixed